rust/processor/src/db/common/models/default_models/parquet_transactions.rs

```diff
@@ -255,6 +274,7 @@ impl Transaction {
                     txn_version,
                     block_height,
                     block_timestamp,
+                    write_set_size_info,
```

Comment:
i don't think you need this here anymore right since it's in parquet_write_set_changes below right?

Reply:
we are passing this to build parquet_write_set_changes. the structure of this parsing is weird. b/c we build everything from transaction